schema/filter_artworks.js

```diff
@@ -48,6 +48,9 @@ const FilterArtworks = {
     dimension_range: {
       type: GraphQLString,
     },
+    extra_aggregation_gene_ids: {
```

Comment:
Not sure i follow. It probably doesn't make sense for one person to work on this part/repo, and another person to work on Force....they're inextricably linked.